src/Simplex/Messaging/Agent/Store/SQLite.hs

```diff
@@ -365,7 +367,11 @@ migrateSchema st migrations confirmMigrations = do
     confirm err = confirmOrExit $ migrationErrorDescription err
     run ms = do
       let f = dbFilePath st
+          fWal = f <> "-wal"
+          fShm = f <> "-shm"
       copyFile f (f <> ".bak")
+      whenM (doesFileExist fWal) $ copyFile fWal (f <> ".bak-wal")
+      whenM (doesFileExist fShm) $ copyFile fShm (f <> ".bak-shm")
```

Comment:
```suggestion
      whenM (doesFileExist fWal) $ copyFile fWal (f <> ".wal.bak")
      whenM (doesFileExist fShm) $ copyFile fShm (f <> ".shm.bak")
```

Reply:
the point was that .bak file is also a valid database with these names